### ISSUE TITLE
Check if new version of BlockMarket is available on launch of application?

### ISSUE THREAD

**Issue by themarketbank** [syscoin/blockmarket-desktop-public#233]
Could BMDT check whether there is a newer version available on launch and provide an option to automatically update it?

Could keep users interested in updates / progress who do not follow on slack.

It may also help with user retention if they can see bugs are fixed quickly and enhanced features added regularly.

**Comment by dwasyluk** [comment]
Yes this is definitely an enhancement I would like to get in as soon as possible as well as auto-downloading of said updates if enabled.
